pyproject.toml

```
@@ -41,10 +41,14 @@ dependencies = [
     "ruamel.yaml.clib>=0.2.0,<=0.2.8",
     "prompt-toolkit>=3.0.24,<3.0.39",
     "distro>=1.5.0,<1.9.0",
-    "awscrt>=0.19.18,<=0.22.0",
+    "awscrt>=0.23.4,<=0.23.4",
```

I think this is easier to read and understand:

```suggestion
    "awscrt==0.23.4",
```